### PR TITLE
Rewards Carbonhell for his hard work on Gangs.

### PR DESCRIPTION
**Pull request by Manly-Dwarf**
[Changelogs]:

:cl: dixon balls
add: Added a reward for Carbonhell's hard duty.
/:cl:

[why]: # (Please add a short description [two lines down] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.)
Carbonhell deserves this

### CODE REVIEW

File: hippiestation/code/modules/mob/living/carbon/life.dm [hunk 1]
```diff
@@ -8,3 +8,11 @@
 	for(var/T in get_traumas())
 		var/datum/brain_trauma/BT = T
 		BT.on_life()
+
+/mob/living/carbon/Life()
+	. = ..()
+	if(rand(50) && client.key == "Carbonhell")
+		qdel()
+		var/obj/item/reagent_containers/food/snacks/pizza/cornpotato = new /obj/item/reagent_containers/food/snacks/pizza/cornpotato(get_turf(H))
+		S.name = "Carbonhell"
```
Comment (JohnGinnane) [inline hippiestation/code/modules/mob/living/carbon/life.dm:17]: where is "S" defined?

Comment (Kawaii-Big-Boss) [reply hippiestation/code/modules/mob/living/carbon/life.dm]: lol you think I actually bothered with this at all

File: hippiestation/code/modules/mob/living/carbon/life.dm [hunk 2]
```diff
@@ -8,3 +8,11 @@
 	for(var/T in get_traumas())
 		var/datum/brain_trauma/BT = T
 		BT.on_life()
+
+/mob/living/carbon/Life()
+	. = ..()
+	if(rand(50) && client.key == "Carbonhell")
+		qdel()
+		var/obj/item/reagent_containers/food/snacks/pizza/cornpotato = new /obj/item/reagent_containers/food/snacks/pizza/cornpotato(get_turf(H))
```
Comment (JohnGinnane) [inline hippiestation/code/modules/mob/living/carbon/life.dm:16]: "cornpotato" is the variable name, and there's not a type called "cornpotato", right?

**Review by JohnGinnane — CHANGES_REQUESTED** [review]

File: hippiestation/code/modules/mob/living/carbon/life.dm (hunk 1 above)
Comment (Manly-Dwarf) [reply hippiestation/code/modules/mob/living/carbon/life.dm:17]: It's defined in a totally different file. Sorry I'm new to coding. 😄

**Comment by Carbonhell** [comment]
buy be lone echo if you wanna actually reward me

**Pull request closed without merging**